### PR TITLE
Generate some stuff for the random ores feature in runtime

### PR DESCRIPTION
**Pull request by JFronny**
Currently a draft because some stuff is still not implemented
EDIT: I am not going to do procedural textures/names in this PR so this PR is finished

### CODE REVIEW

File: src/main/java/fr/anatom3000/gwwhit/materials/CustomOre.java [hunk 1]
```diff
@@ -107,10 +158,229 @@ public void onInitialize() {
...
+        addRecipe("block", JRecipe.shaped(
+                JPattern.pattern("MMM", "MMM", "MMM"),
+                JKeys.keys().key("M", JIngredient.ingredient().item(materialId.toString())),
+                JResult.result(blockId.toString())
+        ));
+        addRecipe(type.toString(), JRecipe.shapeless(
+                JIngredients.ingredients().add(JIngredient.ingredient().item(blockId.toString())),
+                JResult.itemStack(material, 9)
+        ));
+        if (rnd.nextInt(10) == 0) {
```
Comment (C10udburst) [inline src/main/java/fr/anatom3000/gwwhit/materials/CustomOre.java:288]: Maybe pass the dye color as an argument? The dye recipes weren't random, but based on ore color.

Comment (JFronny) [reply src/main/java/fr/anatom3000/gwwhit/materials/CustomOre.java:288]: This is probably a good idea, that argument could also be used if someone wants to try to implement texture generation (the only non-procedural piece apart from the lang file currently)

Comment (C10udburst) [reply src/main/java/fr/anatom3000/gwwhit/materials/CustomOre.java:288]: i mean when you manage to procedurally generate textures you wouldn't need to pass anything to the class except a seed and just generate the rest

Comment (JFronny) [reply src/main/java/fr/anatom3000/gwwhit/materials/CustomOre.java:290]: Done

**Pull request merged**